portfolio/src/main/webapp/Controller.js

```diff
@@ -0,0 +1,80 @@
+export default class Controller {
+
+  constructor(document) {
+    this.username = undefined;
+    this.loggedIn = false;
+    this.document = document;
+  }
+
+  openNameForm() {
+    // pops up the form with id "nameForm".
+  }
+
+  closeNameForm() {
+    // hides the form with id "nameForm".
+  }
+
+  /**
+   * Loads all comments from backend.
+   * This function is called when the page is loaded.
+   * @param {HTML div object} commentBox The div in document with id "comment-box"
+   */
+  static async loadComments(commentBox) {
```

Comment:
I'm really not sure about whether I should use static functions here, and if so, should I separate them into another class (semantically util.js sounds more reasonable)?

Reply:
Yeah I split codes into separate modules in the newest commit. Thanks!